src/main.js

```diff
@@ -27,8 +27,8 @@ let render = () => {
 // Developer Tools Setup
 // ========================================================
 if (__DEV__) {
-  if (window.devToolsExtension) {
-    window.devToolsExtension.open()
+  if (window.__REDUX_DEVTOOLS_EXTENSION__) {
+    window.__REDUX_DEVTOOLS_EXTENSION__.open()
```

Comment:
Yep, if i recall correctly it was meant to open the devtools window in case the "debug in new window" option was enabled. Good catch @zalmoxisus.